lib/version_utils.pm

```diff
@@ -892,3 +893,13 @@ sub php_version {
     ($php, $php_pkg, $php_ver);
 }
 
+=head2 is_community_jeo
+
+Returns true if jeos/minimal-vm images are maintained by the openSUSE community
+=cut
+
+sub is_community_jeos {
+    return is_jeos && is_opensuse &&
```

Comment:
Just `return get_var('FLAVOR', '') =~ /JeOS-for-(AArch64|RPi);` should be enough. That implies `is_jeos && is_opensuse`